### PR TITLE
Wire pointer buttons for fuchsia

### PR DESCRIPTION
**Pull request by cpizano**
Right now can't tell right from left button.

### CODE REVIEW

**Comment by cpizano** [comment]
I signed it!  I am covered as an employee of Google Inc.

File: content_handler/runtime_holder.cc [hunk 1]
```diff
@@ -686,6 +686,8 @@ void RuntimeHolder::OnEvent(mozart::InputEventPtr event,
     pointer_data.device = pointer->pointer_id;
     pointer_data.physical_x = pointer->x * viewport_metrics_.device_pixel_ratio;
     pointer_data.physical_y = pointer->y * viewport_metrics_.device_pixel_ratio;
+    //Buttons are single bit values starting with kMousePrimaryButton = 1.
```
Comment (abarth) [inline content_handler/runtime_holder.cc]: Missing space between `//` and `Buttons`

**Comment by abarth** [comment]
LGTM

**Comment by abarth** [comment]
Thanks!

**Pull request merged**